### PR TITLE
Some tweaks that might make jenkins happier

### PR DESCRIPTION
**Pull request by mfalesni**
{{pytest: cfme/tests/control/test_actions.py cfme/tests/test_rest.py -v --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 9ec50865424d08bab7b08eca85373b5eb12674e8:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d8d30af23fafc5f361e261bdc89098e3491fcd7d:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./utils/api.py:10:1: F401 'lazycache' imported but unused
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e36a11b624e04865f9f8ebd5f25655c73239d9ba:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/control/test_actions.py [hunk 1]
```diff
@@ -150,7 +145,8 @@ def vm(request, provider, small_template, vm_name):
             pytest.skip("{} is quite likely overloaded! Check its status!\n{}: {}".format(
                 provider.key, type(e).__name__, str(e)))
 
-    def finalize():
+    @request.addfinalizer
+    def _finalize():
```
Comment (seandst) [inline cfme/tests/control/test_actions.py]: Why the underscore here?

Comment (mfalesni) [inline cfme/tests/control/test_actions.py]: I like putting an underscore in front of anything that is really private and should not be touched from outside.

Comment (mfalesni) [inline cfme/tests/control/test_actions.py]: @seandst It should be ok to merge, there are some issues with rhos but I want to resolve that when the provider dry comes in. This should kill off the vsphere6 errors

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit eb53821c877308cb842a2388a4ee005546a34ba2:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1c6e907fca8af7d03fff9bacd0dd8f45bf9eb55f:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 29b3ca8e4f9e29e33c512ee949ebe9ad94f99c30:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0625d127d90cfc1674b5b17a67ae09692b093ea7:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 774e000a8d0dbdeeb999e69ffe063a5b86a6f40d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 8c8999b178261702da4566a368478c37517c55d9:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**